.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,74 @@
+name: "Bug Report"
+description: Report something that is broken or just not working as it should
+title: "[BUG]: "
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **Before You Start...**
+
+        This form is only for submitting bug reports. If you have a usage question
+        or are unsure if this is really a bug, make sure to:
+
+        - Read the [docs](https://github.com/nexus-xyz/nexus-zkvm)
+        - Ask on [Telegram Chat](https://t.me/nexus_zkvm)
+        - Ask on [Github Discussions](https://github.com/nexus-xyz/nexus-zkvm/discussions)
+
+        Also try to search for your issue - it may have already been answered or even fixed in main or a development branch but not yet released.
+
+        However, if you find that an old, closed issue still persists in the latest version, you should open a new issue using the form below instead of commenting on the old issue.
+  - type: textarea
+    id: program-info
+    attributes:
+      label: Program Information
+      description: |
+        Provide information about your zkVM program, ideally a minimal example to replicate the bug.
+      placeholder: Program information
+    validations:
+      required: true
+  - type: textarea
+    id: program-info
+    attributes:
+      label: Project Information
+      description: |
+        Provide information about your project using the Nexus zkVM, ideally a minimal example to replicate the bug.
+    validations:
+      required: true
```

Comment:
isn't this part duplicated?

Reply:
I was trying to distinguish between the zkVM program and the Rust program invoking it (e.g., through an SDK).

Reply:
I'd say if it confused me, then it will certainly confuse external contributors.

Doesn't sound like a big deal though.

Reply:
Added some language to clarify.